tests/integration/operatorhub/cmd_service_test.go

```diff
@@ -51,6 +52,68 @@ var _ = Describe("odo service command tests for OperatorHub", func() {
 			Expect(stdOut).To(ContainSubstring("service can be created/deleted from a valid component directory only"))
 		})
 
+		Context("a namespace specific operator is installed", func() {
```

Comment:
> `a namespace specific operator is installed`

Can we word this statement differently? I'm asking because when I first saw this, it seemed like we're testing that installing an Operator in a specific namespace worked fine. Can we word it to something that makes it more obvious that we're testing parameterized service creation for Operator backed services?

Reply:
that is present here https://github.com/girishramnani/odo/blob/6d2fccc57372e17ef724c09375f119d25b24d83f/tests/integration/operatorhub/cmd_service_test.go#L80
I could be wrong but I have similar approach like here https://github.com/girishramnani/odo/blob/6d2fccc57372e17ef724c09375f119d25b24d83f/tests/integration/operatorhub/cmd_service_test.go#L117
Where the context explains what is done and then internal `when` and `it` explains whats done after the `Context's` description is done